core/server/worker/src/main/java/alluxio/worker/netty/DataServerUFSBlockReadHandler.java

```diff
@@ -35,8 +35,8 @@
  * {@link DataServerReadHandler}.
  */
 @NotThreadSafe
-final class DataServerUfsBlockReadHandler extends DataServerReadHandler {
-  private static final Logger LOG = LoggerFactory.getLogger(DataServerUfsBlockReadHandler.class);
+final class DataServerUFSBlockReadHandler extends DataServerReadHandler {
```

Comment:
It was favored in the package but it seems like `Ufs` is favored in the codebase, updated thanks.